pymc/distributions/continuous.py

```diff
@@ -3242,7 +3246,7 @@ def logcdf(value, lower, c, upper):
 
         Parameters
         ----------
-        value: numeric or np.ndarray or aesara.tensor
+        value : tensor_like of float
             Value(s) for which log CDF is calculated. If the log CDF for multiple
             values are desired the values must be provided in a numpy array or Aesara tensor.
```

Comment:
Every `logcdf( )` and `logp( )` functions are taking more than one parameter but we have the documentation for only one do we need to add doc for others as well?